index.js

```diff
@@ -16,6 +16,10 @@ var through2 = require('through2'),
       cmd: 'npm',
       args: ['install']
     },
+    'package.json#yarn': {
```

Comment:
I think it would be more straightforward to just pass in a map to package.json

```js
'package.json': {
  npm: {
    cmd: 'npm',
    args: ['install']
  },
  pnpm: {
    cmd: 'pnpm',
    args: ['install']
  },
  yarn: {
    cmd: 'yarn',
    args: []
  }
}
```

yes, and I think it would be nice to add [pnpm](github.com/rstacruz/pnpm) and [ied](github.com/alexanderGugel/ied) as well.